ArcadiaParties.CQRS/Handlers/GetAssistantUserHandler.cs

```diff
@@ -13,15 +13,14 @@ public class GetAssistantUserHandler : IRequestHandler<GetAssistantUserQuery, As
     {
         private readonly IAssistantTokenRepository _tokenRepository;
         private readonly IHttpClientFactory _clientFactory;
+        private const string requestUser = "https://assistant.arcadia.spb.ru/api/user";
```

Comment:
Constants should be placed at the very beginning of the class, before all other members. It is common code style.

Reply:
Replaced